public/200.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+  <head>
```

Comment:
Add 200.html in build folder after running `npm run build` so that the routes do not show 404.